### DISCUSSION TITLE
Connect to local node through IPC

### DISCUSSION THREAD

**Discussion by aamargant** [NomicFoundation/hardhat#3424] [Troubleshooting] [answered]
How do you connect to a local node through Inter-process Communications with hardhat config? you just put the IPC path on the url key?

`const config: HardhatUserConfig = {
  networks: {
    myLocalNode: {
      url: "/home/<your-username>/.bor/data/bor.ipc",
      chainId: 0x89,
      accounts: [deployer.private],
    }}}`

Like this?

**Comment by fvictorio** [comment, marked as answer]
Hi @aamargant, that's not supported at the moment. Feel free to open an issue with that feature request. I don't think it's something we'll be able to do soon, but having an issue is useful to keep track on how much interest there is from other users.

**Reply by SwapnilSoni1999** [reply]
any insights on this?
